payjoin-cli/src/main.rs

```diff
@@ -14,7 +15,11 @@ fn main() -> Result<()> {
     match matches.subcommand() {
         Some(("send", sub_matches)) => {
             let bip21 = sub_matches.get_one::<String>("BIP21").context("Missing BIP21 argument")?;
-            app.send_payjoin(bip21)?;
+            let fee_rate_sat_per_vb = sub_matches.get_one::<f32>("fee_rate").unwrap_or(&2.1);
+            let fee_rate_sat_per_kwu = fee_rate_sat_per_vb * 250.0_f32;
+            let fee_rate: bitcoin::FeeRate =
+                bitcoin::FeeRate::from_sat_per_kwu(fee_rate_sat_per_kwu.ceil() as u64);
+            app.send_payjoin(bip21, fee_rate)?;
```

Comment:
Since the BIP21 Uri is parsed from String inside send_payjoin, I think it would be good form to do the same parsing FeeRate from f32 with this feerate parameter

Reply:
moved to `send_payjoin` and added log in the cli to indicate we are using default `fee_rate` in case its not provided